test/scripts/build-image

```diff
@@ -26,7 +26,7 @@ def main():
         print(json.dumps(config, indent=2))
         config_name = config["name"]
 
-    testlib.runcmd(["go", "build", "-o", "./bin/build", "./cmd/build"])
+    testlib.runcmd(["go", "build", "-buildvcs=false", "-o", "./bin/build", "./cmd/build"])
```

Comment:
Do you remember what the actual error was? This seems weird, since the binary is being built inside a Git repository checkout, so things should work. And the `go help build` says that the command should error out only when using `-buildvcs=true`. The default is `-buildvcs=auto`.